<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Employee</class>
 <widget class="QDialog" name="Employee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="Employee_tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Employee_backButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Employee_deleteButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Employee_insertButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="insert_line">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="delete_line">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Employee_searchButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_line">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
